### DISCUSSION TITLE
提问

### DISCUSSION THREAD

**Discussion by PureDimension** [liusy58/NJU_AdvancedProgramming_SP22#191] [Q&A] [answered]
1.地图导入，只要实现”给定txt，可以通过读取该txt生成对应地图“就可以，不需要实现类似”地图编辑器“之类的功能是吗？
2.我个人的塔防是太空主题的，带火都会飞，设定里的”飞行单位“在我这里等价转换成”量子隐形单位“可以吗？
3.要求中”远程攻击的”敌人和“能攻击远程单位的“敌人能否合并？个人的塔防里分星球上和太空中两种环境，敌人的路径在太空中，部分敌人设计成只能攻击太空中的友军，而太空中的友军大多数只能近战，但是两者无绝对关系（存在牵引反重力炮台等太空远程单位），想知道这么设计会不会把这部分的分给吃掉emm

**Comment by liusy58** [comment, marked as answer]
1. 可以
2 3. 没太明白，可以Q戳我
